.github/workflows/industrial_ci_action.yml

```diff
@@ -0,0 +1,100 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - IMAGE: 'foxy-ci'
+            CCOV: true
+            AFTER_RUN_TARGET_TEST: './.ci.prepare_codecov'
+          - IMAGE: 'foxy-ci-testing'
+            BEFORE_TARGET_TEST_EMBED: 'set +u && source moveit_kinematics/test/test_ikfast_plugins.sh && set -u'
+            CLANG_TIDY: true
+    env:
+      DOCKER_IMAGE: moveit/moveit2:${{ matrix.env.IMAGE }}
+      UPSTREAM_WORKSPACE: moveit2.repos
+      CCOV: ${{ matrix.env.CCOV }}
```

Comment:
This is confusing. Just refer to matrix.env.CCOV everywhere.
```suggestion
```

Reply:
It isn't always defined, which causes the if statements to fail in the steps.  I could set it to false for every not codecov job then I wouldn't need this.

Reply:
That is what I remembered why I did that, I pushed this change though and it seems to be working so far.

Reply:
It looks like it worked, this change is now in.